projects/ccd-case-ui-toolkit/src/lib/shared/components/case-viewer/services/case.resolver.ts

```diff
@@ -85,22 +85,27 @@ export class CaseResolver implements Resolve<CaseView> {
           this.caseNotifier.announceCase(this.caseNotifier.cachedCaseView);
           return this.caseNotifier.cachedCaseView;
         }),
-        catchError(error => this.checkAuthorizationError(error))
+        catchError(error => this.checkAuthorizationError(error, cid))
       ).toPromise();
   }
 
-  private checkAuthorizationError(error: any) {
+  private checkAuthorizationError(error: any, caseReference: string) {
+    console.error(error);
```

Comment:
@johnbenjamin-hmcts - please remove this `console.error()` statement if no longer required

Reply:
@Daniel-Lam I removed this `console.error` statement.